iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestTables.java

```diff
@@ -181,23 +181,28 @@ public Table createTable(TestHiveShell shell, String tableName, Schema schema, P
         "'" + TableProperties.DEFAULT_FILE_FORMAT + "'='" + fileFormat + "')");
 
     if (records != null && !records.isEmpty()) {
-      StringBuilder query = new StringBuilder().append("INSERT INTO " + identifier + " VALUES ");
-
-      records.forEach(record -> {
-        query.append("(");
-        query.append(record.struct().fields().stream()
-                .map(field -> getStringValueForInsert(record.getField(field.name()), field.type()))
-                .collect(Collectors.joining(",")));
-        query.append("),");
-      });
-      query.setLength(query.length() - 1);
-
-      shell.executeStatement(query.toString());
+      String query = getInsertQuery(records, identifier, false);
+      shell.executeStatement(query);
     }
 
     return loadTable(identifier);
   }
 
+  String getInsertQuery(List<Record> records, TableIdentifier identifier, boolean isOverwrite) {
```

Comment:
nit: Since it is used in a test then I might make the method `public`

Reply:
Done